pkg/structs/flamebearer/html.go

```diff
@@ -26,9 +28,8 @@ func FlamebearerToStandaloneHTML(fb *FlamebearerProfile, dir http.FileSystem, w
 	scriptTpl, err := template.New("standalone").Parse(
 		`
 <script type="text/javascript">
-	try {
-		eval('window.flamegraph = {{ .Flamegraph }}');
-	} catch(e) {}
+	window.flamegraph = {{ .Flamegraph }}
```

Comment:
@eh-am I figured the eval here was not necessary so I removed it. But maybe I'm missing something, let me know.